web/satellite/src/components/header/projectSelection/ProjectDropdown.vue

```diff
@@ -22,6 +22,11 @@
             >
                 <p class="project-dropdown__wrap__choice__unselected">{{ project.name }}</p>
             </div>
+						<div @click="toggleSelection" @click.prevent.stop="closeDropdown" class="project-dropdown__wrap__create-project__wrapper">
```

Comment:
indents are broken here. 
we should have only one `@click` action here. You can put closeDropdown() call inside of toggleSelection() call.

Reply:
If we have this button here then we should remove Create Project button from app header entirely. And if we do that, then we should have separate PR removing this button in tardigrade-branding repo